src/current/v24.1/upgrade-cockroach-version.md

```diff
@@ -276,34 +283,36 @@ Once you are satisfied with the new version:
     > RESET CLUSTER SETTING cluster.preserve_downgrade_option;
     ~~~
 
+    A series of migration jobs runs to enable certain types of features and changes in the new major version that cannot be rolled back. These include changes to system schemas, indexes, and descriptors, and enabling certain types of improvements and new features. Until the upgrade is finalized, these features and functions will not be available and the command `SHOW CLUSTER SETTING version` will return `{{ previous_version_numeric }}`.
```

Comment:
`{{ previous_version_numeric }}` is still displaying `v23.2` and not `23.2`. If it needs to be the latter, then I figure the variable has to be fixed somehow?

Reply:
Here, the problem was that the variable I wanted is actually `page.version.version` on these pages, despite being `page.major_version` on normal pages.

Reply:
It wasn't fixed by replacing `strip_first` with `remove_first`? Anyway, seems to be working now.